services/app-web/src/testHelpers/apolloMocks/rateDataMock.ts

```diff
@@ -118,6 +118,7 @@ const rateRevisionDataMock = (data?: Partial<RateRevision>): RateRevision => {
             certifyingActuaryContacts: [
                 {
                     __typename: 'ActuaryContact',
+                    id: '123-cert-actuary',
```

Comment:
how come you added the ID to contacts?

Reply:
@macrael I don't remember now something with the types matching and determining which contact was which when there's multiple additional actuaries. Is this something you want changed?